src/components/manage/Widgets/DatetimeWidget.jsx

```diff
@@ -122,6 +122,48 @@ export class DatetimeWidgetComponent extends Component {
     return this.props.dateOnly || this.props.widget === 'date';
   }
 
+  renderMonthElement = ({ month, onMonthSelect, onYearSelect }) => {
+    let i;
+    let years = [];
+    for (i = this.moment().year(); i >= this.moment().year() - 100; i--) {
```

Comment:
I don't think we should assume the widget is being used to select a date in the past. Maybe default to showing 100 years in the past + 20 in the future, but also provide widget options to customize the range?